labels.yaml

```diff
@@ -1,285 +1,336 @@
+- name: kind/technical-debt
+  color: c7def8
+- name: kind/upgrade-test-failure
+  color: fbca04
+- name: lgtm
+  color: 15dd18
+- name: needs-ok-to-merge
+  color: ededed
+- name: needs-ok-to-test
+  color: b60205
+- name: needs-rebase
+  color: BDBDBD
+- name: non-release-blocker
+  color: 0e8a16
+- name: ok-to-merge
+  color: fbca04
+- name: priority/awaiting-more-evidence
+  color: fef2c0
+- name: priority/backlog
+  color: fbca04
+- name: priority/critical-urgent
+  color: e11d21
+- name: priority/failing-test
+  color: e11d21
+- name: priority/important-longterm
+  color: eb6420
+- name: priority/important-soon
+  color: eb6420
+- name: priority/P0
+  color: ff0000
+- name: priority/P1
+  color: ededed
+- name: priority/P2
+  color: ededed
+- name: priority/P3
+  color: ededed
+- name: queue/blocks-others
+  color: ffaa00
+- name: queue/critical-fix
+  color: ffaa00
+- name: queue/fix
+  color: ffaa00
+- name: queue/multiple-rebases
+  color: ffaa00
+- name: release-blocker
+  color: d93f0b
+- name: release-note
+  color: c2e0c6
+- name: release-note-action-required
+  color: c2e0c6
+- name: release-note-label-needed
+  color: db5a64
+- name: release-note-none
+  color: c2e0c6
+- name: requires-release-czar-attention
+  color: d93f0b
+- name: retest-not-required
+  color: eb6420
+- name: retest-not-required-docs-only
+  color: fbca04
+- name: sig/api-machinery
+  color: d2b48c
+- name: sig/apps
+  color: d2b48c
+- name: sig/auth
+  color: d2b48c
+- name: sig/autoscaling
+  color: d2b48c
+- name: sig/aws
+  color: d2b48c
+- name: sig/big-data
+  color: d2b48c
+- name: sig/cli
+  color: d2b48c
+- name: sig/cluster-lifecycle
+  color: d2b48c
+- name: sig/cluster-ops
+  color: d2b48c
+- name: sig/contributor-experience
+  color: d2b48c
+- name: sig/docs
+  color: d2b48c
+- name: sig/federation
+  color: d2b48c
+- name: sig/instrumentation
+  color: d2b48c
+- name: sig/network
+  color: d2b48c
+- name: sig/node
+  color: d2b48c
+- name: sig/onprem
+  color: d2b48c
+- name: sig/openstack
+  color: d2b48c
+- name: sig/rktnetes
+  color: d2b48c
+- name: sig/scalability
+  color: d2b48c
+- name: sig/scheduling
+  color: d2b48c
+- name: sig/service-catalog
+  color: d2b48c
+- name: sig/storage
+  color: d2b48c
+- name: sig/testing
+  color: d2b48c
+- name: sig/windows
+  color: d2b48c
+- name: size/L
+  color: ee9900
+- name: size/M
+  color: eebb00
+- name: size/S
+  color: 77bb00
+- name: size/XL
+  color: ee5500
+- name: size/XS
+  color: "009900"
+- name: size/XXL
+  color: ee0000
+- name: stale
+  color: "795548"
+- name: team/api (deprecated - do not use)
+  color: ededed
+- name: team/cluster (deprecated - do not use)
+  color: ededed
+- name: team/control-plane (deprecated - do not use)
+  color: ededed
+- name: team/gke
+  color: d2b48c
+- name: team/huawei
+  color: d2b48c
+- name: team/mesosphere
+  color: d2b48c
+- name: team/redhat
+  color: d2b48c
+- name: team/test-infra
+  color: ededed
+- name: team/ux (deprecated - do not use)
```

Comment:
I'd like to see them gone too, but wanted to keep this PR trivial.  I defer to @kubernetes/sig-contributor-experience-misc for modification of labels since I don't have write privs here.  Unclear to me whether removing, renaming, or swapping labels would alter github issue recency.